doc/cephfs/fs-volumes.rst

```diff
@@ -149,10 +150,10 @@ Create a subvolume group using::
 The command succeeds even if the subvolume group already exists.
 
 When creating a subvolume group you can specify its data pool layout (see
-:doc:`/cephfs/file-layouts`), uid, gid, file mode in octal numerals and
+:doc:`/cephfs/file-layouts`), uid, gid, file mode in octal numerals, and
```

Comment:
The Oxford comma today, the Oxford comma tomorrow, the Oxford comma forever!

Reply:
Damn straight.